File: Psets/pset1/Mario/mario.c
#include<stdio.h>
#include<cs50.h>


int Print(void);
int main(void)
{
    int x = Print(); //initialization function
    for (int i = 0; i < x; i++)  // for loops specifying number of spaces and hashes per line
    {
        for (int j = 0; j < x; j++)
        {
            if (i + j < x - 1)
                printf(" ");
            else
                printf("#");
        }
        printf("\n");
    }
}
int Print(void) //start the function
{
    int H;

    // prompts user for pyramid height between 1 and 8
    do
    {
        H = get_int("Enter The Hieght Between 1 AND 8 : ");
    }
    while (H <= 0 || H > 8); //make the hieght between 1 and 8
    return H;
}

File: Psets/pset6/mario/less/mario.py
from cs50 import get_int


def main():

    # initialization function
    x = Print()
# for loops specifying number of spaces and hashes per line
    for i in range(0, x, +1):
        for j in range(0, x, +1):
            if (i + j) < (x - 1):
                print(" ", end="")
            else:
                print("#", end="")
        print()


def Print():
    # print function
    while True:
        H = get_int("Enter the Height: ")
        if H > 0 and H <= 8:
            return H


main()
